buzsaki_lab_to_nwb/tingley_code_septal/tingleyseptalbehaviorinterface.py

```diff
@@ -163,3 +164,26 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict):
                     )
             [table.add_row(**row) for row in sorted(data, key=lambda x: x["start_time"])]
             processing_module.add(table)
+
+        # Add epochs
+        lfp_file_path = session_path / f"{session_path.name}.lfp"
+        raw_file_path = session_path / f"{session_id}.dat"
+        xml_file_path = session_path / f"{session_id}.xml"
+
+        if raw_file_path.is_file():
```

Comment:
@CodyCBakerPhD  your comments in slack make me thing that maybe there is a more standard way of doing this?  I am not sure that we can rely (can we?) on the recorders already being instantiated from the other data interfaces (as they are in the same `run_conversion` method as here in the behavioral interface) so I just went an extracted the data directly from the source.

Reply:
@h-mayorquin In a more general context, yeah, I would have suggested handling this at the `NWBConverter.__init__` class level, after the `super()` initializes all the objects. (For `RecordingExtractor` objects you can also set these things at the object level via these functions: https://github.com/SpikeInterface/spikeextractors/blob/master/spikeextractors/baseextractor.py#L363

However, I think the way you're doing it here is just fine and deals with the addition in a safe, reliable manner.